pkg/services/accesscontrol/roles.go

```diff
@@ -0,0 +1,115 @@
+package accesscontrol
+
+import "github.com/grafana/grafana/pkg/models"
+
+var PredefinedRoles = map[string]RoleDTO{
+	usersSelfRead: {
+		Name:    usersSelfRead,
+		Version: 1,
+		Permissions: []Permission{
+			{
+				Permission: "users:read",
```

Comment:
```suggestion
				Permission: ActionUsersRead,
```

Reply:
Oh, I forgot to remove this thing. It's not used anywhere, so I'm removing it instead of applying these changes.